### DISCUSSION TITLE
How to pair bluetooth?

### DISCUSSION THREAD

**Discussion by Beavxx** [nakhonthai/ESP32APRS_T-TWR#41] [General]
Hello everyone.

Great work here!!  I can't seem to figure out how to pair Bluetooth.  What is the procedure?  Bluetooth is on however none of my devices see the TWR.  

Thank you!!!

**Comment by derekmurawsky** [comment]
You need to boost the bluetooth power up. In the device's bluetooth menu (not via the web app, but the device itself) you can set the power level. Setting that up will let your other device find it. 
I still haven't been able to connect, though. I get a pairing error with an invalid passcode/passkey, but I'm never prompted to enter one. I tried emptying the filed in the bluetooth config on the device, but it didn't seem to work. Any thoughts?

**Reply by derekmurawsky** [reply]
I noticed that the security callbacks were commented out? Could that be why there is an issue?
https://github.com/nakhonthai/ESP32APRS_T-TWR/blob/0d690c9b45a6acb78f75c04733dc7e1777ddb2c6/src/main.cpp#L157
